nautobot/core/forms/fields.py

```diff
@@ -781,8 +781,10 @@ class TagFilterField(DynamicModelMultipleChoiceField):
     """
 
     def __init__(self, model, *args, query_params=None, queryset=None, **kwargs):
+        from nautobot.extras.models import Tag
+
         if queryset is None:
-            queryset = model.tags.all()
+            queryset = Tag.objects.filter(content_types=ContentType.objects.get_for_model(model))
```

Comment:
I forget - can we do `Tag.objects.get_for_model(model)` instead?

Looks like evaluating `ContentType.objects.get_for_model()` here is no bueno when migrations haven't been run yet.

Reply:
Interesting, thanks for the help!